network-api/networkapi/tests.py

```diff
@@ -465,3 +467,136 @@ def test_software_page_export_fields(self):
         self.assertIn("Host controls", export_fields)
         self.assertIn("Easy to learn and use", export_fields)
         self.assertIn("Easy to learn and use help text", export_fields)
+
+
+class TestCloudinaryMigration(TestCase):
+
+    def setUp(self):
+        pni_homepage = BuyersGuidePageFactory.create(
```

Comment:
let's `from django.conf import settings` and then make setUp return if `USE_CLOUDINARY` isn't `True`.